tests/char_volume_test.cpp

```diff
@@ -44,19 +44,18 @@ TEST_CASE( "character_baseline_volumes", "[volume]" )
     Character &you = get_player_character();
     REQUIRE( you.get_mutations().empty() );
     REQUIRE( you.height() == 175 );
-    CHECK( you.get_base_volume() == 70_liter );
 
     REQUIRE( your_height_with_trait( trait_SMALL2 ) == 70 );
-    CHECK( your_volume_with_trait( trait_SMALL2 ) == 4480_ml );
+    CHECK( your_volume_with_trait( trait_SMALL2 ) == 23326_ml );
 
     REQUIRE( your_height_with_trait( trait_SMALL ) == 122 );
-    CHECK( your_volume_with_trait( trait_SMALL ) == 23717_ml );
+    CHECK( your_volume_with_trait( trait_SMALL ) == 42476_ml );
 
     REQUIRE( your_height_with_trait( trait_LARGE ) == 227 );
-    CHECK( your_volume_with_trait( trait_LARGE ) == 152778_ml );
+    CHECK( your_volume_with_trait( trait_LARGE ) == 116034_ml );
 
     REQUIRE( your_height_with_trait( trait_HUGE ) == 280 );
-    CHECK( your_volume_with_trait( trait_HUGE ) == 286720_ml );
+    CHECK( your_volume_with_trait( trait_HUGE ) == 156228_ml );
```

Comment:
I'm concerned that this is further separating character volume from monster volume by quite a bit. The *minimum* volume to a monster to be HUGE is 483.76L. After this, it seems the **average** HUGE character volume is merely 156.23L, or about a third of volume.

This isn't a blocker, but I think it's worth pointing out.

Reply:
I see. I'm not familiar with how the HUGE trait is meant to work. Physics wise, assuming similar densites around 1kg/L, a HUGE monster (483L) would weigh around 500 kg too. Depending on how realistic we want to be, we could also adjust densities depending on the monster in the future (say, a bloated zombie filled with gas would be less dense than a kevlar hulk).
Game play wise, it would be nice to make monsters and the character behave similarly with the same physical constrains where possible.